apps/staking/lib/constants.ts

```diff
@@ -286,6 +287,12 @@ export const prefDetails = {
     defaultValue: false,
     description: 'Show awaiting operator contracts in the open nodes page',
   },
+  [PREFERENCE.V2_Rewards]: {
+    label: 'Use V2 Rewards Endpoint',
+    type: 'boolean',
+    defaultValue: false,
+    description: 'Use the V2 rewards endpoint',
+  },
```

Comment:
TODOLATER: If users are interacting with preferences maybe we should start localizing them?